docs/ide/default-keyboard-shortcuts-in-visual-studio.md

```diff
@@ -53,6 +53,7 @@ The sections in the following table include commands that are global in that you
 
 |Commands|Keyboard shortcuts|
 |--------------| - |
+|Build.BuildSelection|**Ctrl+B**|
```

Comment:
```suggestion
|Build.BuildSelection|**Ctrl+B** (Visual Studio 2019)|
```

Reply:
@gewarren, okay to merge now as is?

Reply:
@jborsecnik No, I'd like that "(Visual Studio 2019)" bit to be added since that shortcut doesn't apply to VS 2017.